src/libexpr/eval.hh

```diff
@@ -477,6 +477,9 @@ struct EvalSettings : Config
 
     Setting<bool> useEvalCache{this, true, "eval-cache",
         "Whether to use the flake evaluation cache."};
+
+    Setting<bool> traceVerbose{this, false, "trace-verbose",
+        "Whether `builtins.traceVerbose` should trace its first argument when evaluated."};
```

Comment:
While I did add this in my branch, I'm not sure if we want this. On one hand, having `-v` also have an influence on evaluation is a bit weird, but on the other hand it probably won't be a problem.

Reply:
Yeah, I think it's fine. Does not really matter.